aptos-move/block-executor/src/executor.rs

```diff
@@ -49,6 +49,11 @@ use std::{
     sync::{atomic::AtomicU32, Arc},
 };
 
+struct InternalExecuteResult {
+    record_succeeded: bool,
+    updates_outside: bool,
```

Comment:
What does `updates_outside` mean?
Comments would be helpful.

Reply:
I'll actually change this to Result<bool, PanicError> in a PR that handles fallback_to_sequential, so leaving this for now